http/src/client/mod.rs

```diff
@@ -49,6 +48,11 @@ impl Debug for State {
     }
 }
 
+pub enum ReactionType {
```

Comment:
This should go in request/channel/reaction/mod since it's only used there

Reply:
On second thought we probably really do need some other type of name though... I'm usually not against re-using names but this is awfully confusing since they are similar contexts